docs/readthedocs/models/bm_model.md

```diff
@@ -7,7 +7,7 @@ The BM model is part of the SIMONA Simulation framework and represented by an ag
 
 ### Parameters
 
-![](../_static/figures/models/bm_model/bm_parameters.png)
+![](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/ie3-institute/simona/dev/docs/readthedocs/_static/figures/models/bm_model/bm_parameters.png)
```

Comment:
Please check, that only uml diagrams send through the proxy of plantuml. Here the input of a .png leads to an error.

Reply:
I checked all the diagrams and fixed the problems.